medical_center/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Medical Centers",
+    "summary": "Adds a concept of Medical Centers to Patients.",
+    "version": "10.0.1.0.0",
+    "category": "Medical",
+    "website": "https://laslabs.com",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "medical",
+    ],
+    "data": [
+        "views/medical_center.xml",
+        "views/medical_patient.xml",
+        "views/medical_menu.xml",
```

Comment:
There's actually a debate about this one right now, so we're in somewhat of a state of limbo. I'm on the side of adding the suffix though, which is inline with current docs. I recommend you state your feelings on the subject in maintainer-tools#191 😄 

For now @kellylougheed please make these files follow the guidelines, adding a `_view`, `_demo`, or `_data` suffix where applicable please and thanks!